cmd/markdown/support.md

```diff
@@ -0,0 +1,29 @@
+# Support
+
+## Community Resources
+
+Need help? Join the [Atmos community!](https://cloudposse.com/slack)  
+
+- **Slack Community**: Connect with active users in the `#atmos` channel and get help anytime.  
+- **GitHub Discussions**: Post questions or join conversations with the community and Cloud Posse. 
+- **Issue Tracker**: Found a bug? Report it [here](https://github.com/cloudposse/atmos/issues).  
+
+## Office Hours
+
+We hold **office hours every Wednesday at 11:30 AM PST**. Join us for live Q&A, pairing sessions, or help with debugging issues.  
+
+## Paid Support  
+
+We offer priority support to GitHub Sponsors (Enterprise Tier).
+
+[](https://github.com/sponsors/cloudposse)
```

Comment:
Curious how this looks. What I don't like about the empty link is that it adds a blank space in front of the link.

```suggestion
[🔗](https://github.com/sponsors/cloudposse)
```